examples/standalone/benchmarks/plotting.py

```diff
@@ -0,0 +1,52 @@
+import json
+import os
+from argparse import ArgumentParser
+from datetime import datetime
+
+import matplotlib.pyplot as plt
+
+
+if __name__ == "__main__":
+    usage = "usage: python %(prog)s <data_dir>"
+    parser = ArgumentParser(usage=usage)
+    parser.add_argument(
+        "data_dir",
+        type=str,
+        action="store",
+        help="directory containing data to run with",
+    )
+    args = parser.parse_args()
+
+    # collect and sort the data
+    alldata = []
+    for filename in os.listdir(args.data_dir):
+        if filename.endswith(".json"):
+            print(filename)
+            fullpath = args.data_dir + "/" + filename
+            with open(fullpath) as f:
+                alldata.append(json.load(f))
+    alldata.sort(
+        key=lambda k: datetime.strptime(
+            k["setup"]["experiment time"], "%d/%m/%Y %H:%M:%S"
+        )
+    )
+
+    # simple plots for the history
+    for data in ["main", "init", "cleanup", "total"]:
```

Comment:
is the plan to expand 'main' to include finer scale metrics matching what Oli added to the Fortran code?

Reply:
yeah, this is what @oelbert is working on